redisvl/providers/huggingface.py

```diff
@@ -0,0 +1,29 @@
+from typing import Dict, List, Optional
+
+from sentence_transformers import SentenceTransformer
```

Comment:
we could consider importing this scoped within the class -- and then we can prevent folks from having to install SentenceTransformers (torch, etc) if they don't plan to need it?

Reply:
It's the default for LLMCache... should we not assume that? I do like the scoped gaurd approach.